src/core/Form/Combobox/ChipList/ChipList.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { HtmlDiv, HtmlDivProps } from '../../../../reset';
+import { AutoId } from '../../../../utils/AutoId';
+import { baseStyles } from './ChipList.baseStyles';
+
+const baseClassName = 'fi-chip-list';
+
+const chipListClassNames = {
+  wrapper: `${baseClassName}_wrapper`,
+};
+
+export interface ChipListProps extends HtmlDivProps {
+  className?: string;
```

Comment:
Is there a reason why className is not documented?

Reply:
No specific reason. 
Might have been just left out as the prop is not visible to outside/user.

Can copy&paste the same JSDoc for this to be in line with others.

Reply:
Added JSDoc for the `className` prop.